recipes/lexbor/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+import functools
+
+required_conan_version = ">=1.43.0"
+
+class LexborConan(ConanFile):
+    name = "lexbor"
+    description = "Lexbor is development of an open source HTML Renderer library"
+    topics = ("html5", "css", "parser", "renderer")
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lexbor/lexbor/"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False],
+        "build_separately": [True, False],
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+        "build_separately": False,
+    }
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", ]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_msvc(self):
+        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
+
```

Comment:
```suggestion
```

No longer needed. Now you can use https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc directly, because the CI is running Conan 1.45.0 now

Reply:
@uilianries  
Thank you. I don't know these features.